uaclient/messages/__init__.py

```diff
@@ -1772,6 +1772,13 @@ def __repr__(self):
     + ENABLE_FAILED,
 )
 
+E_REPO_PURGE_FAIL_NO_ORIGIN = FormattedNamedMessage(
+    "repo-purge-fail-no-origin",
+    "Cannot disable {entitlement_name} with purge: no origin value defined"
+    + "\n"
+    + DISABLE_FAILED_TMPL,
+)
```

Comment:
This isn't an error per se since it isn't used in an exception. Other CanDisable messages are in the "Named Messages" section above the error section

Reply:
will move